src/components/QuickOpenModal.js

```diff
@@ -356,6 +356,24 @@ export class QuickOpenModal extends Component<Props, State> {
     });
   };
 
+  shouldShowErrorEmoji() {
+    const { query } = this.props;
+
+    if (this.isGotoQuery()) {
+      return !/^:\d*$/.test(query);
+    }
+
+    if (
+      this.isVariableQuery() ||
+      this.isFunctionQuery() ||
+      this.isShortcutQuery()
+    ) {
+      return !this.getResultCount();
```

Comment:
yeah - that's not a bad idea. 

We could also remove this ` !this.getResultCount();` check because it should be redundant